### ISSUE TITLE
Provide example on how to create a field.

### ISSUE THREAD

**Issue by waynehamadi** [twentyhq/twenty#5518]
<img width="1093" alt="Screenshot 2024-05-21 at 6 43 20 PM" src="https://github.com/twentyhq/twenty/assets/9652976/bd33e12e-6ada-416e-8748-553c3cf76a0c">

As you can see the object is empty right now:
https://docs.twenty.com/rest-api/metadata#/operations/createOneField

**Comment by waynehamadi** [comment]
<img width="454" alt="Screenshot 2024-05-21 at 6 57 59 PM" src="https://github.com/twentyhq/twenty/assets/9652976/c09dbc94-6d9b-4299-9599-0e3491785866">

and the default empty object provided by the doc doesn't work:
{
  "statusCode": 400,
  "message": "Unknown type \"CreateOneFieldInput\". Did you mean \"CreateFieldInput\", \"DeleteOneFieldInput\", \"CreateOneObjectInput\", \"CreateObjectInput\", or \"UpdateFieldInput\"?",
  "error": "Bad Request"
}

**Comment by charlesBochet** [comment]
I have investigated this issue, it seems that we don't have a consistent naming across our /metadata API regarding fields and objects.

Current namings:
- CreateOneFieldMetadataInput
- UpdateOneFieldMetadataInput
- DeleteOneFieldInput
- CreateOneObjectInput
- UpdateOneObjectInput
- DeleteOneObjectInput

The REST resolver is trying to compute these Input types but is lost as they are not consistent.

We should update the naming in the graphql /metadata API and then update the REST /metadata API.

I would vote for CreateOneFieldInput to be consistent with the naming of the graphql API objects / fields. (we do store objects as ObjectMetadata but we have accepted to keep the public API without the metadata key word as there is no possible confusion with ObjectRecords there)

FYI @ijreilly regarding Metadata API scope and @martmull regarding REST API
